### ISSUE TITLE
Final review by Wenchang Yang

### ISSUE THREAD

**Issue by SSTXNEO** [TSL-123/SentimentDrivenStrategy#19]
This project is trying to predict the effects of news on trading markets. Specifically, the project tries to analyze the sentimental information from news, and then simulate the trading process using the models' results to see whether the sentimental information helps to make a significant difference on the return.

The idea of this project is convincing and well-founded. Broadly speaking, this project might be able to help financial researchers and practitioners learn sentimental information from news to reach wiser investment decisions. I personally like the idea to use sentimental information extracted from real news to help simulate the real-world trading process. The models this group picked seemed to show a very convincing and well-executed results about how their methods work. 

Besides the way the group extracted sentimental information and simulate trading process, I also have some concerns about the data set they were using: the group mentioned that they used a financial dictionary consisting of 2337 negative words and 353 positive words. Since there are significantly more negative features in determining the overall sentimental information of a sentence, can we expect to see more news be assigned as "negative news" than those that would be assigned as "positive words"? Moreover, can we expect to see a possible bias of prediction on the sentimental information about news towards negative side? Having these questions answered in the report might help readers have a better understanding about how the project deal with the uneven distribution of numbers of sentimental features. 

To summarize, this project is very throughout and well-executed. It gave a reasonable model result and practical implication on the trading research. The group might want to try to consider further develop their algorithms and integrate them as a Python package to make them applied for more purposes.
